### ISSUE TITLE
Track down unwanted `text-transform: uppercase` in the block editor 

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/bobbingwide#77]
In jetpack.wp.a2z I've noticed that there are a number of instances of inline CSS with `text-transform: uppercase`.
This is making it difficult to use the block editor, especially when the text being transformed is an input field and the content was originally created in mixed case. 

![image](https://user-images.githubusercontent.com/2474435/191024681-419f8134-4012-40ec-8359-78a1df45f282.png)
In this example we see the text of the Jetpack Consent field being uppercased.
The inline style for the control's label is being applied to the Jetpack field label's contents.
```
.css-1v57ksj {
    font-size: 11px;
    font-weight: 500;
    line-height: 1.4;
    text-transform: uppercase;
    display: inline-block;
    margin-bottom: calc(8px);
    padding: 0px;
}
```
Deselecting the `text-transform: uppercase` in the inspector we see the text in the case it was originally written.
Here you can see that I'd edited it and typed an uppercase N to match what I could see.

Can we seNd you an email from time to time?

Also the labels in the Settings panels are being uppercased: eg Size, Default and Radius

With Gutenberg:
![image](https://user-images.githubusercontent.com/2474435/191022792-a1ae01d5-cee7-44a2-bba8-2bdd3645c591.png)

Without Gutenberg:
![image](https://user-images.githubusercontent.com/2474435/191023062-0238418b-5e4b-44d4-86e5-ecf8c2b410d8.png)

It's probably the same issue.

**Comment by bobbingwide** [comment]
The code that's causing the problem is in https://github.com/WordPress/gutenberg/blob/167fac279339b37c28f73b56824eaae581a6a20b/packages/components/src/utils/base-label.ts

**Comment by bobbingwide** [comment]
Note that the `text-transform: uppercase` applies in the block editor. On the front end it's OK... apart from the accidental editing of the content with UPPERCASE characters. 
See https://jetpack.wp-a2z.org/block/consent-jetpack-field-consent/

**Comment by bobbingwide** [comment]
The uppercasing affects other blocks. Here's a screenshot of a Checkbox in CoBlock's **Request a Quote** variation of the Form block. 
![image](https://user-images.githubusercontent.com/2474435/192286196-56785150-0e34-4641-bf43-ab1615da98c8.png)

**Comment by muhme** [comment]
I have the same unwanted UPERCASE on my block control labels. And also the [block editor tutorial](https://developer.wordpress.org/block-editor/getting-started/tutorial/) shows the problem with "STARTING YEAR" in the editor for label property “Starting year”.

**Comment by bobbingwide** [comment]
https://github.com/bobbingwide/bobbingwide/issues/77#issuecomment-1886359355

I don't think we'll be able to do anything about this with regards to the block editor. I found out what was happening. So now we just have to live with it.

**Comment by muhme** [comment]
Yes, the labelling of the controls should be in capital letters, see https://github.com/WordPress/gutenberg/issues/42782, so I'll leave it at that - thanks.
